src/services/users.js

```diff
@@ -40,9 +43,11 @@ const createNewUser = async (user) => {
 	}
 	const userProfile = rows[0]
 	const body = { id: userProfile.id, email: userProfile.email }
-	const token = jwt.sign({ user: body }, config("TOKEN_SECRET"))
+	const accessToken = jwt.sign({ user: body }, config("TOKEN_SECRET"),
+		{expiresIn: '900s'})
+	const refreshToken = crypto.randomBytes()
```

Comment:
the generated refresh token should be saved in the database

Reply:
so you can generate it before doing the user insert

Reply:
yes please